shell/shell_test.go

```diff
@@ -45,21 +45,21 @@ func TestMenderShellExecShell(t *testing.T) {
 	}
 
```

Comment:
do you think we could add here `[]string{""}` scenario? also, maybe we should test a scenario with bad parameters, what happens then?

Reply:
Good idea! Fixed!